internal/server/data/deviceflowauthrequest.go

```diff
@@ -58,35 +58,52 @@ func CreateDeviceFlowAuthRequest(tx WriteTxn, dfar *models.DeviceFlowAuthRequest
 	return insert(tx, (*deviceFlowAuthRequestTable)(dfar))
 }
 
-func GetDeviceFlowAuthRequest(db GormTxn, selectors ...SelectorFunc) (*models.DeviceFlowAuthRequest, error) {
-	return get[models.DeviceFlowAuthRequest](db, selectors...)
+type SelectDeviceFlowAuthRequestOptions struct {
```

Comment:
So far we've always named these options struct to match the function that accepts them

```suggestion
type GetDeviceFlowAuthRequestOptions struct {
```

any reason this one needs to be different?